### PR TITLE
first version of index page

### PR DESCRIPTION
**Pull request by tomzacchia**

### CODE REVIEW

**Comment by tomzacchia** [comment]
Screen Shot 2019-09-19 at 4 04 49 PM

File: main.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  background: url('./assets/hero.jpg') 40% 100% no-repeat;
+  height: 90vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: white;
+  font-weight: 100;
+}
```
Comment (maelfosso) [inline main.css]: Please add 
```css
background-size: cover;
```

**Comment by tomzacchia** [comment]
@maelfosso I have made the appropriate changes however I did not add the empty h1 tag. What would be the point in adding empty elements? Seems like a contradiction to best practices.

**Comment by tomzacchia** [comment]
Icon placement is as such on 1440 x 683 screen: 
Screen Shot 2019-09-20 at 3 15 36 PM

**Pull request merged**